src/status_im2/contexts/profile/create/events.cljs

```diff
@@ -28,22 +20,3 @@
            :password           (native-module/sha3 (security/safe-unmask-data password))
            :imagePath          (profile.config/strip-file-prefix image-path)
            :customizationColor color})})
-
-(rf/defn login-new-profile
```

Comment:
hm

Reply:
it's not used as far as I can see?

Reply:
yeah, just wondering why, probably i forgot to remove it during refactoring